### ISSUE TITLE
Feature Request - More Payment Methods

### ISSUE THREAD

**Issue by gordebak** [itchio/itch.io#598]
Hello,

Neither PayPal nor Stripe work in my country. I'd love to see more payment methods, preferably direct ones.

Thanks.

**Comment by gordebak** [comment]
I could make PayPal work. But an easier method for me is preferable. Thanks.

**Comment by p0nce** [comment]
Hi There.

I would move from Gumroad to itch.io in a heardbeat if direct payment methods (like cards) were available.
Itch.io is superior to Gumroad in every way I can think of, except this.

**Comment by Deozaan** [comment]
@gordebak I'm just a passerby, but maybe this suggestion would be more useful if you said what country you are in, and gave an example or two of payment methods that do work for you and your country.

**Comment by gordebak** [comment]
@Deozaan, you're right, thank you. I live in Turkey, and neither Paypal nor Stripe work in my country. I'd love it if there were a direct-to-card payment method.

**Comment by fasterthanlime** [comment]
> gave an example or two of payment methods that **do** work for you and your country.

**Comment by asudyn** [comment]
I saw that other marketplaces usually have wire transfers (via SWIFT system). This thing has pretty big fees (like $20 to $50 depending on bank and amount), and you can't transfer small amounts, it has to be more than $250. 
I've researched a bit and it looks like Skrill may be the best option here. They work worldwide. Their fees aren't that big. They have some problems but it looks like it's the best option for third world countries right now. Freelancer.com uses Skrill as one of the payment methods.
There's also Payoneer. They send you a MasterCard (costs $10) and allow third parties to send money to it for 2% fee and $30 per year. Airbnb uses Payoneer as one of the payment methods. It's a very popular method but I'm not sure if everyone can send money to your card or a company has to partner with Payoneer for this. I would be nice if Itch.io could partner with Payoneer. If it's worth it, of course.

**Comment by quantumproducer** [comment]
+1 for Paypal - a player just asked why they couldn't use Paypal on itch.io

**Comment by leafo** [comment]
@quantumproducer PayPal already works on itch.io, are you having a trouble with a buyer not being able to purchase your game?

**Comment by quantumproducer** [comment]
@leafo , oh I expected for Itch to receive money through Paypal then payout via bank transfer. I do not have a paypal account setup for itch which is why it's not an option for the player, I think?

<img width="458" alt="screen shot 2017-09-10 at 12 28 24" src="https://user-images.githubusercontent.com/11827932/30250859-c545df22-9623-11e7-9699-ca5c8b9dafa2.png">

**Comment by quantumproducer** [comment]
@leafo what are your thoughts on this? Should I setup a corporate paypal as the way to receive paypal payments? I've been iffy about paypal, I need to see if I can configure to only pay-out to my bank and not be able to take money from.

**Comment by leafo** [comment]
@quantumproducer For direct payments we're phasing out PayPal since the API is severely limited and doesn't let us to basic things like do refunds. If you want to support PayPal buyers then I recommend using out payouts mode.

**Comment by victorleblais** [comment]
Sorry to comment on a two-year old issue, but there wasn't anything done regarding accepting more payment options.

I'm not concerned since I can use Paypal or direct payment, but there are tons of people that can't use either.

Adding support for [Skrill](https://www.skrill.com/) and/or [PaySafeCard](https://www.paysafecard.com/) would really go a long way and would open up Itch to a lot more potential customers.

**Comment by gingerbeardman** [comment]
- Wise
- Revolut

**Comment by Novack** [comment]
Another nice thing, would be to have cryptocurrency payments that do not require Stripe. In my country Stripe is not supported, thus I lose both, Stripe itself and crypto.

**Comment by Djdefrag** [comment]
Revolut could be a great addition :)
